apps/server/src/shared/domain/entity/team.entity.ts

```diff
@@ -33,11 +33,19 @@ export class Team extends BaseEntityWithTimestamps {
 	name: string;
 
 	@Embedded(() => TeamUser, { array: true })
-	userIds: TeamUser[];
+	private userIds: TeamUser[];
```

Comment:
isn't it more useful to rename the field to "teamUsers" and map it via "fieldName" instead of making it private and create a getter/setter?

Reply:
It would be nice, but fieldName does not exist in EmbeddedOptions and Embedded mixed up with the property annotation isnt supported. You will not notice the getter. It works like a normal property call :)

Reply:
fieldName does not exist on Embedded